NAMESPACE

```diff
@@ -121,22 +117,33 @@ import(circlize)
 import(config)
 import(cowplot)
 import(data.table)
-import(dbplyr)
 import(digest)
 import(dplyr)
+import(forcats)
 import(g3viz)
 import(ggbeeswarm)
 import(ggplot2)
 import(ggpubr)
 import(ggrepel)
 import(ggsci)
 import(ggthemes)
+import(glue)
 import(grid)
+import(gridExtra)
 import(maftools)
 import(metaviz)
 import(parallel)
 import(plotly)
+import(purrr)
+import(readr)
+import(reshape2)
 import(rtracklayer)
-import(tidyverse)
+import(ssh)
+import(stats)
```

Comment:
which package imports stats? We should drop this because it introduces conflicts with function names and gives lots of warnings on package installation/importing

Reply:
Agreed. Assuming you're meaning "what function* imports stats", these are the functions that are currently importing stats;

```
FtestCNV
prettyForestPlot
prettyOncoplot
```

Reply:
For this, the code is still using functions from the `stats` package (from my testing).

- FtestCNV calls `fisher.test` and `adjust.p`
- prettyForestPlot calls `adjust.p`
- prettyOncoplot calls `setNames`